ops/charm.py

```diff
@@ -335,13 +513,43 @@ class CharmEvents(ObjectEvents):
 class CharmBase(Object):
     """Base class that represents the Charm overall.
 
-    Usually this initialization is done by ops.main.main() rather than Charm authors
-    directly instantiating a Charm.
+    CharmBase is used to create a charms. This is done by inheriting
+    from CharmBase and customising the sub class as required. So to
+    create your own charm, say "MyCharm" define a charm class and
+    setup the required event handlers (hooks) in its constructor as
+    shown below
+
+    ```
```

Comment:
need to check but i suspect this way of formatting the comment doesn't work

Reply:
Ok.Removed triple back quotes and indented entire code block by 4 spaces